src/Microsoft.DocAsCode.MarkdigEngine/Validation/ValidationExtension.cs

```diff
@@ -25,6 +28,13 @@ public void Setup(MarkdownPipelineBuilder pipeline)
 
             pipeline.DocumentProcessed += document =>
             {
+                if (InclusionContext.IsInclude 
+                && (string.Equals(Path.GetExtension(InclusionContext.RootFile?.ToString()), ".yml", StringComparison.OrdinalIgnoreCase) 
+                || string.Equals(Path.GetExtension(InclusionContext.RootFile?.ToString()), ".yaml", StringComparison.OrdinalIgnoreCase)))
+                {
+                    document.SetData("rootFileYamlMime", YamlMime.ReadMime(InclusionContext.RootFile?.ToString()));
```

Comment:
```suggestion
                    document.SetData("SchemaName", YamlMime.ReadMime(InclusionContext.RootFile?.ToString()));
```

Reply:
+1

---
In reply to: [339871951](https://github.com/dotnet/docfx/pull/5234#discussion_r339871951) [](ancestors = 339871951)

Reply:
Agree